WordPress/WordPressUITests/Screens/Me/SupportScreen.swift

```diff
@@ -0,0 +1,45 @@
+import UITestsFoundation
+import XCTest
+
+// This screen object is for the Support section. In the app, it's a modal we can get to from Me > Help & Support, or, when logged out, from Prologue > tap either continue button > Help.
+
+private struct ElementStringIDs {
+    static let closeButton = "close-button"
+    static let helpCenter = "help-center-link-button"
+    static let contact = "contact-support-button"
+    static let myTickets = "my-tickets-button"
+    static let contactEmail = "set-contact-email-button"
+    static let activityLogs = "activity-logs-button"
+}
+
+class SupportScreen: BaseScreen {
+    let closeButton: XCUIElement
+    let helpCenter: XCUIElement
+    let contact: XCUIElement
+    let myTickets: XCUIElement
+    let contactEmail: XCUIElement
+    let activityLogs: XCUIElement
+
+    init() {
+        let app = XCUIApplication()
+        closeButton = app.buttons[ElementStringIDs.closeButton]
+        helpCenter = app.cells[ElementStringIDs.helpCenter]
+        contact = app.cells[ElementStringIDs.contact]
+        myTickets = app.cells[ElementStringIDs.myTickets]
+        contactEmail = app.cells[ElementStringIDs.contactEmail]
+        activityLogs = app.cells[ElementStringIDs.activityLogs]
+
+        super.init(element: activityLogs)
+
+        //Check that each item is loaded:
+        XCTAssert(helpCenter.exists)
+        XCTAssert(contact.exists)
+        XCTAssert(myTickets.exists)
+        XCTAssert(contactEmail.exists)
+        XCTAssert(activityLogs.exists)
```

Comment:
Not really a suggestion, more a "did you know that", so no need to update this code.

Did you know that you can create a list of objects and run the same code on each, to avoid writing repetitive code like this?

```swift
[helpCenter, contact, myTickets, contactEmail, activityLogs].forEach { XCTAssert($0.exists) }
```

Reply:
Actually! After having said that I realized that, in this case, it's a bad suggestion 😳 🤦‍♂️ .

Explicitly writing each assertion means that if one fails, Xcode will highlight it. With the `forEach`, we'd get the failure in `XCTAssert($0.exist)` but that doesn't tell us which of the element didn't exist!

Reply:
Oh yeah, I remember learning .forEach in JS! I think you're right that it's best to leave them separated in this case, but this is a good reminder to me.